src/cmdc/datasets/official/base.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import textwrap
+
+from cmdc.datasets import OnConflictNothingBase
+
+
+class CountyData(OnConflictNothingBase):
+    table_name = "us_covid"
+    pk = '("vintage", "dt", "fips", "variable_id")'
+
+    def __init__(self):
+        super(CountyData, self).__init__()
+
+        return None
+
+    def _insert_query(self, df, table_name, temp_name, pk):
+        out = f"""
+        INSERT INTO data.{table_name} (vintage, dt, fips, variable_id, value)
+        SELECT tt.vintage, tt.dt, tt.fips, mv.id as variable_id, tt.value
+        FROM {temp_name} tt
+        LEFT JOIN meta.covid_variables mv ON tt.variable_name=mv.name
+        ON CONFLICT {pk} DO NOTHING
+        """
+
+        return textwrap.dedent(out)
+
+    def get(self):
+
+        df = pd.read_csv(CA_COUNTY_URL)
+        df = df.rename(columns=C_RENAMER).loc[:, list(C_RENAMER.values())]
+        df["dt"] = pd.to_datetime(df["dt"])
+        df["vintage"] = pd.datetime.today()
+
+        # Create totals
+        df["hospital_beds_in_use_covid_total"] = df.eval(
```

Comment:
Do we need any checks that these columns exist? 

what if the total one does, but the other two don't? Will this line overwrite the total with NaN?

Reply:
This whole get method should have been deleted when I decided to abstract it out. Will address this in your other comment.